mypy/treetransform.py

```diff
@@ -67,12 +67,38 @@ def visit_import_from(self, node: ImportFrom) -> Node:
     def visit_import_all(self, node: ImportAll) -> Node:
         return ImportAll(node.id, node.relative)
 
+    def copy_argument(self, argument: Argument) -> Argument:
+        init_stmt = None  # type: AssignmentStmt
+
+        if argument.initialization_statement:
+            init_lvalue = cast(
+                NameExpr,
+                self.node(argument.initialization_statement.lvalues[0]),
+            )
+            init_lvalue.set_line(argument.line)
+            init_stmt = AssignmentStmt(
```

Comment:
Could we just have `__init__` of `Argument` construct us a new init statement? We can trust that it always does the right thing. Now we have two implementations of init statement creation.